app/controllers/go_controller.rb

```diff
@@ -0,0 +1,39 @@
+class GoController < ApplicationController
+  before_action :enforce_valid_game_key, only: [:show, :join]
+
+  helper_method :key
+
+  layout "go"
+
+  def create
+    game = Go::Game.create
+
+    # TODO: Games persist in memory for 1 hour, remove when shift to DB persistence
+    GoGameCleanupJob.set(wait: 1.hour).perform_later(key: game.key)
```

Comment:
this might be a silly idea, we don't have any job backend (but probably don't need one for this little demo site) but I wanted to have something that could clean up the memory just in case. was thinking about adding a clock to say how much time is left using the periodic timer, but probably more trouble than its worth. maybe i'll add something to the view that's like "this game will expire at X:XX UTC"